src/webgpu/api/validation/encoding/encoder_state.spec.ts

```diff
@@ -19,4 +19,67 @@ TODO:
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
```

Comment:
Also I know the TODO at the top of this file is confusing but we should try to update it as best as possible since we're adding some of the tests it describes.

Reply:
ok, let me update the description in the following CL because I feel like the updated description should be reviewed as well.